content/applications/inventory_and_mrp/inventory/management/products/usage.rst

```diff
@@ -50,15 +50,17 @@ PACK", do this for each package.
 .. image:: media/usage_06.png
     :align: center
 
-Packaging
-=========
+Packagings
+==========
 
-The packaging is the physical container that protects your product. If
-you are selling computers, the packaging contains the computer with the
-notice and the power plug.
+The packaging is a standard container containing several units of your 
+product. If you are selling cans of soda, the packagings would be a 
+6-Pack, a 15-Pack or even a pallet.
 
-In Odoo, packagings are used for indicative purposes on sale orders.
-They can be specified on the product form, in the inventory tab.
+In Odoo, packagings are used for indicative purpose on sales/purchase
```

Comment:
Just a small edit, but in line 60 could you use `In Odoo, packagings are used as an indicative purpose on sales/purchase`
 for better flow?

Reply:
done